html/pfappserver/root/graph/dashboard.tt

```diff
@@ -195,840 +250,309 @@ END
-                        </div>
-                        [% END %]
-                     </div>
-                </div>
+            
+            <div class="card">
+              <div class="card-title">
+                <h2>[% l('NTLM') %]</h2>
+              </div>
+              <div class="card-block">
+                <div class="row-flex">
+                  [% graph("statsd_timer_ntlm_auth.time", "NTLM latency", {"filter_graph" => "average"}) | none %]
+                  [% graph("statsd_counter_ntlm_auth.failures", "NTLM failures", {"filter_graph" => "counter"}) | none %]
+                </div>
+              </div>
             </div>
         </div>
+
         <div class="tab-pane[% IF tab == "authentication" %] active[% END %]" id="authentication">
             [% IF sources.size > 0 %]
             <div class="card">
-                <div class="card-title">
-                    <h2>[% l('Authentication Sources') %]</h2>
-                </div>
-                <div class="card-block">
-                    <div class="row-flex">
-                    [%
-                        SET sourcesSize = sources.size;
-                        IF sourcesSize == 1;
-                            SET spanSize = gridSize;
-                        ELSIF sourcesSize MOD 3 == 0;
-                            SET spanSize = gridSize / 3;
-                        ELSE;
-                            SET spanSize = gridSize / 2;
-                        END
-                    %]
-                        [% FOREACH source IN sources %]
-                        [% FOREACH host IN source.host.split(',') %]
-                        <div class="col[% spanSize %] text-center">
-                            <div data-netdata="fping.[% host.replace('\.', '_') %]_latency"
-                            data-host="/netdata/127.0.0.1"
-                            data-common-max="ping"
-                            data-common-min="ping"
-                            data-title="[% source.description %] - ping [% host %]"
-                            data-chart-library="dygraph"
-                            data-colors="[% palette(loop.index) %]"
-                            data-height="200px"
-                            data-hide-missing="true"
-                            data-before="0"
-                            data-after="-86400"></div>
-                        </div>
-                        [% END %]
-                        [% END %]
-                    </div>
+              <div class="card-title">
+                <h2>[% l('Authentication Sources') %]</h2>
+              </div>
+              <div class="card-block">
+                <div class="row-flex">
+                  [% FOREACH source IN sources %]
+                    [% FOREACH host IN source.host.split(',') %]
+                    [% graph("fping." _ host.replace('\.', '_') _ "_latency", source.description _ "- ping " _ host, {"nocluster" => 1}) | none %]
```

Comment:
I propose to restore the logic surrounding the graphs width: full width when only one source is defined, a third when a multiple of three, a half otherwise.

Reply:
Using thirds creates unreadable graphs on some screens and relying only on halfs creates uneven rows so that's why I put full rows for this.